question5.txt

```diff
@@ -1 +1,5 @@
+5. How to set multiple remote repositories for the same project? Explain the use case. you should have a working demo of it.
+
+
+
 to set multiple remote repositaries in git for the same project i can simply use the git remote add origin command and add my repositaries from github or other platform like gitlab.
```

Comment:
This answer is partially correct.
The actual answer is:
You can add multiple repositories for same project on different platforms like: github, gitlab etc.
That can be done as below:
```powershell

git remote add origin <repository-url>

git remote add gitlab <repository-url>

git remote -v

git push origin main

git push gitlab main

```

Reply:
And this question is asking for working demo, so you can add commands for whole process, like from remote add, commit to branch, push to origin and gitlab, and then pull etc